knative-operator/hack/manifestival.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/knative-operator/vendor/github.com/manifestival/manifestival/client.go b/knative-operator/vendor/github.com/manifestival/manifestival/client.go
+index a8d108ce..85994972 100644
+--- a/knative-operator/vendor/github.com/manifestival/manifestival/client.go
++++ b/knative-operator/vendor/github.com/manifestival/manifestival/client.go
```

Comment:
Is it worth to apply upstream? and eventually use a release containing the patch ?

Reply:
I think as soon as we get the k8s deps straight, we get rid of this patch anyway. It's more a 'backwards' patch (to make manifestival compatible with older k8s), so don't think upstream is keen on downgrading their k8s deps :)

Reply:
(this is how I understoo @markusthoemmes )

Reply:
Exactly, we only need this because of our hilariously old k8s deps.